pkg/controllers/management/k3sbasedupgrade/deployPlans.go

```diff
@@ -30,7 +31,7 @@ func (h *handler) deployPlans(cluster *v3.Cluster, isK3s, isRke2 bool) error {
 	)
 	switch {
 	case isRke2:
-		upgradeImage = rke2upgradeImage
+		upgradeImage = settings.PrefixPrivateRegistry(rke2upgradeImage)
```

Comment:
Can we do the same thing for `k3supgradeImage` on Line 40?

Reply:
@thedadams  
I thought about that, too 😄  
I did not add it because 1) it is out of the scope of the linked issue, and 2) it will change the current behavior for importing k3s clusters.  

How about this: I will make a new issue and a PR for the imported k3s cluster for easier tracking and QA?  
Can we get this PR merge fisrt?